packages/response-ops/rule_form/src/rule_form.tsx

```diff
@@ -60,7 +93,26 @@ export const RuleForm = (props: RuleFormProps) => {
         }
       />
     );
-  }, [id, ruleTypeId, plugins, onCancel, onSubmit]);
+  }, [
+    http,
+    i18n,
+    theme,
+    userProfile,
+    application,
+    notifications,
+    charts,
+    settings,
+    data,
+    dataViews,
+    unifiedSearch,
+    docLinks,
+    ruleTypeRegistry,
+    actionTypeRegistry,
+    id,
+    ruleTypeId,
+    onCancel,
+    onSubmit,
+  ]);
```

Comment:
The entire `CoreStart` object is passed in here, so I destructured the plugins that are needed for the `RuleFormPlugins` interface, since this is in a `useMemo`. React watches these dependencies to rerender the `RuleForm` component when they change.